lib/utilities/paginator.rb

```diff
@@ -0,0 +1,85 @@
+module Utilities
+  module Paginator
+    def items_to_display(item:, items_per_page:)
```

Comment:
Item is misleading.... It makes me think this will be an instance, instead of a class